### DISCUSSION TITLE
PrimeNg TreeTable Feature Requests

### DISCUSSION THREAD

**Discussion by critchie** [primefaces/community#1166] [PrimeNG]
I like the tree table component and it is perfect for what I need in my app. But it lacks some features that the regular table component has. Is it possible to add these features:

1. Advanced filtering
![Screenshot 2024-02-09 224414](https://github.com/primefaces/community/assets/340046/a3254029-e9d7-4076-9f98-6ce0749ec6af)
2. Save state (stateful)

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback, we'll wait for community upvotes to implement this feature.

**Comment by chaitanyaironman** [comment]
Is this feature taken into consideration
